app/dao/jobs_dao.py

```diff
@@ -142,3 +142,59 @@ def dao_get_all_letter_jobs():
     return db.session.query(Job).join(Job.template).filter(
         Template.template_type == LETTER_TYPE
     ).order_by(desc(Job.created_at)).all()
+
+
+@statsd(namespace="dao")
+def dao_get_job_statistics_for_job(service_id, job_id):
+    query = Job.query.join(
+        JobStatistics, Job.id == JobStatistics.job_id
+    ).filter(
+        Job.id == job_id
+    ).join(
+        Template, Template.id == Job.template_id and Template.version == Job.template_version
```

Comment:
Why are we joining onto template? we don't appear to be using any columns from it